public class Exception12 {
    public static void main (String[] args) throws ArithmeticException
    {
        //try {
            int a = 23/0;

        //}
//        catch (Exception e) {
//            System.out.println (e.getMessage ());
//        }

        System.out.println (a);
    }
}